packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -94,7 +94,8 @@
 
                 &.dc-dropdown__display-placeholder--is-title {
                     transform: translate(0, -1.7rem) scale(0.75);
-                    padding: 0 4px;
+                    padding: 0 2px;
+                    width: 112%;
```

Comment:
Is it possible to make width `fit-content` instead?

Reply:
Ok so what I did I removed the width and did a no wrap on span. Please review and approve. Thanks:)